sdk/identity/ci.yml

```diff
@@ -30,6 +30,8 @@ extends:
       LiveTestCtestRegex: azure-identity.
       LineCoverageTarget: 95
       BranchCoverageTarget: 56
+      # TODO: Revert before merge
+      UseFederatedAuth: true
```

Comment:
Just double checking, do we intend to revert this before merging? If so, when does `UseFederatedAuth` get set to true?

Reply:
If identity tests can use federated auth when this is merged then this can stay in place.

Reply:
Tests don't pass so this will be reverted for now. 

https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3925356&view=logs&j=e732a657-4392-5bc3-3289-b34fe17fd560&t=28d3e977-de2c-5feb-854f-4d9592e2dd8b